requirements-optional.txt

```diff
@@ -8,7 +8,7 @@ Flask==2.2.3  # https://github.com/oegedijk/explainerdashboard/issues/259
 fairlearn==0.7.0  # For check_fairness method
 
 # Models
-xgboost>=1.1.0
+xgboost>=1.1.0; python_version != "3.11" or platform_system != "Windows"
```

Comment:
Is it worth to mention in the documentation, that support for those libraries is not guaranteed to work on Windows? I guess it will all be fine on Anaconda, but we currently can't verify it.

Note to self: Add a GHA recipe based on Mamba.